eng/pipelines/libraries/build-job.yml

```diff
@@ -5,6 +5,7 @@ parameters:
   archType: ''
   framework: netcoreapp
   isOfficialBuild: false
+  useLiveCoreClr: false
```

Comment:
Going forward when should this ever be false and what would the impact of that be?

Reply:
In this change I haven't yet dropped the support for both "live-live runs" and "partial runs" picking up some artifacts from external locations. I can easily imagine this limitation may be temporary but, in its current form, dropping this condition would invalidate the "legacy" runtime-libraries pipeline. I assume it's up to further discussion in our meetings how exactly to sequence the transformation.